docs/03-extensions/magento1/reference/axios-instances.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: "Axios instances"
+sidebar_position: 1
+description: "This reference explains how to create `axios` instances to interact with Magento's API."
+---
+
+<p>{frontMatter.description}</p>
+
+
+## `makeAdminClientFromRequest`
+
+The `makeAdminClientFromRequest` function creates an `axios` instance that can be
+used to interact with Magento's API as an Admin.
+
+```js
+makeAdminClientFromRequest(request);
+```
+
+Arguments:
+
+| Name      | Type    | Description                  |
+| :-------- | :------ | :--------------------------- |
+| `request` | Request | v2 compatible request object |
+
+Example:
+
+```js
+import { makeAdminClientFromRequest } from "@front-commerce/magento1/axios";
+
+export default {
+  namespace: "Acme/Feature",
+  contextEnhancer: ({ req }) => {
+    const adminAxiosInstance = makeAdminClientFromRequest(req);
+
+    // rest of logic
+  },
+};
+```
+
+## `makeAuthServiceFromRequest`
+
+The `makeAuthServiceFromRequest` function creates a service that expose related session infos
```

Comment:
**issue:** "Info" / "information" is always singular. Also the sentence is missing a ".".

```suggestion
The `makeAuthServiceFromRequest` function creates a service that expose related session info.
```

**suggestion(alt):** I'd be more favorable to using no abbreviation on such words in the documentation.

```suggestion
The `makeAuthServiceFromRequest` function creates a service that expose related session information.
```

Reply:
fixed